### ISSUE TITLE
NSClient errors a LOT in 2.6 train.

### ISSUE THREAD

**Issue by PedanticAvenger** [MilosKozak/AndroidAPS#2517]
Seems to be due to connection watchdog conns/2min exceeding 5.
Occurs all the damn time. ;)  But specific event at 1:10PM in logs
2.6.0
I have no idea WHY it keeps tripping, I switch between cell/wifi no problem
[AndroidAPS_LOG_1584559040505.log.zip](https://github.com/MilosKozak/AndroidAPS/files/4350924/AndroidAPS_LOG_1584559040505.log.zip)

**Comment by MilosKozak** [comment]
NSClient is disconnecting. It's related to internet connectivity. There is nothing I can do for you

**Comment by PedanticAvenger** [comment]
Right, I move all over all day long, including into/out of underground parkades, can knobs be added to nsclient for thresholds and/or alerting behavior?  i.e. alert if you can't reach nightscout for 30 min or the like?
